sonar-ws/src/main/java/org/sonarqube/ws/client/BaseService.java

```diff
@@ -56,8 +58,10 @@ protected WsResponse call(WsRequest request) {
 
   public <T extends Message> T convert(WsResponse response, Parser<T> parser) {
     try (InputStream byteStream = response.contentStream()) {
+      byte[] bytes = IOUtils.toByteArray(byteStream);
+      System.out.println("BaseService.convert " + bytes.length + " bytes : " + Arrays.toString(bytes));
```

Comment:
any reason to change this beside logging?
if not, modifications to this class should be removed